dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -3,7 +3,8 @@
  * FILE:            dll\win32\getuname\lang\ro-RO.rc
  * PURPOSE:         Unicode Description Strings
  * PROGRAMMERS:     Baruch Rutman (peterooch at gmail dot com)
- * TRANSLATOR:      Ștefan Fulea (stefan dot fulea at mail dot com)
+ * TRANSLATORS:     Ștefan Fulea (stefan dot fulea at mail dot com)
+ *                  Andrei Miloiu (miloiuandrei@gmail.com)
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
```

Comment:
Is the translation update expected, or you just wanted to update authors?

Reply:
I want both of them. Thank you for fixing the issue! Now, I'm going to upload new translated lines.